### ISSUE TITLE
Missed a colon in InfoGAN code

### ISSUE THREAD

**Issue by back2yes** [znxlwm/pytorch-generative-model-collections#8]
Great job. By the way, I found this line of code doubtful: 

https://github.com/znxlwm/pytorch-generative-model-collections/blob/5207b2fda3a9832c182e27dbbf2658b3078021ef/infoGAN.py#L78

Maybe this line should be changed into: 
a = F.sigmoid(x[:, :+1: self.output_dim])

**Comment by dyyzhmm** [comment]
Agreed, but ... the  self.output_dim should always be 1, so, there is no difference with or without the missing colon.
